keyboards/pegasushoof/keymaps/citadel/readme.md

```diff
@@ -0,0 +1,75 @@
+#Citadel keymap for pegasushoof
+
+Designed for ISO-DE and similar ISO layouts.
+
+The central idea of this keymap is to avoid unergonomic key combos that force the user to either press
+multiple keys with one hand or to use the left hand for right-sided keys. For instance, ISO-DE users
+have to press `AltGr`+`7` to type _{_. The keymap allows ergonomic access to the right-sided keys of the
+German AltGr-layer by turning `Caps Lock` into a hybrid Fn/AltGr-key. It can easily be adapted for other
+ISO layouts.
+
+The _Function_ layer also offers mouse emulation and function/media keys in the F-Row.
+
+`Pause/Break` is used as a safe _close key_: A _single tap_ does nothing, a _double tap_ executes ALT+F4.
+
+Named after the homonymous space station from _Mass Effect_. Made by [Daniel Rose](https://github.com/droseger/).
+
+##[1] Layers
+
+###Layer 1 (Default)
+    .----.   .-------------------. .-------------------. .-------------------. .--------------.
```

Comment:
This isn't formatted correctly.  Make sure that you have a double space at the end of each line.